pages/api/revalidate.js

```diff
@@ -11,19 +11,13 @@ export const config = {
 };
 
 const revalidateLookup = {
-  // keys are sourced from clearviction-sanity's repo --> schema folder --> file --> name
+  // keys are sourced from cv-sanity repo -> schema folder -> index -> export schema types
```

Comment:
might be more clear to replace the `->` syntax with just giving pathing `/clearviction-sanity/schemas/index.ts`

Reply:
Hahaha yeah that does seem like a bit easier 😂